draft-ietf-rtcweb-jsep.xml

```diff
@@ -2030,10 +2030,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <xref target="RFC3264"></xref>, Section 5.1.</t>
 
               <t>If this m= section is for media with configurable
-              frame sizes, e.g. audio, an "a=maxptime" line, indicating
-              the smallest of the maximum supported frame sizes out of
-              all codecs included above, as specified in
-              <xref target="RFC4566"></xref>, Section 6.</t>
+              durations of media per packet, e.g., audio, an
+              "a=maxptime" line, indicating the maximum amount of
+              media, specified in milliseconds, that can be
+              encapsulated in each packet, as specified in <xref
+              target="RFC4566"></xref>, Section 6.  This value is set
+              to the smallest of the maximum values across all the
```

Comment:
It's not totally obvious what "maximum values" refers to. Maybe "maximum packetization values"?